tap_absolute_software/streams.py

```diff
@@ -16,6 +16,7 @@ class DeviceStream(AbsoluteSoftwareStream):
     path = "/devices-advanced"
     primary_keys = ["deviceUid"]
     replication_key = None
+    records_jsonpath = "$.data[*]"
```

Comment:
Since `records_jsonpath` is part of the `AbsoluteSoftwareStream` base class, we don't need it here
We would only need to specify it here if a particular API returned a result that didn't follow the standard pattern, but these do